workspaces/bookmarks/plugins/bookmarks/src/index.ts

```diff
@@ -20,6 +20,7 @@
  * @packageDocumentation
  */
 
+export { default } from './alpha';
```

Comment:
So you want export it from the main package? Most plugins exports the NFS still as /alpha.

You can do it differently of course, but what do you think about exporting it here and as /alpha?

But fyi, we asked this recently different authors and at the moment most of the plugins follows this convention.